userspace/libscap/examples/01-open/test.c

```diff
@@ -183,6 +407,17 @@ int main(int argc, char** argv)
 
 		if(res != SCAP_TIMEOUT)
 		{
+			if(ev->type == evt_type)
+			{
+				lens16 = (uint16_t*)((char*)ev + sizeof(struct ppm_evt_hdr));
+				valptr = (char*)lens16 + ev->nparams * sizeof(uint16_t);
+				printf("\n------------------ EVENT: %d TID:%lu\n", evt_type, ev->tid);
```

Comment:
Can we print event name instead of its id here?

Reply:
Maybe we can do that, but since it is for `developers` we can just keep the `event_id` to be clearer